dev-requirements.txt

```
@@ -1,10 +1,13 @@
+###################
+# SESSION TESTING #
+###################
+
+nox==2021.10.1
```

```suggestion
nox==2021.10.1
```

Could just be this tbh :P